src/sanitizer.js

```diff
@@ -527,6 +523,31 @@ export function resolveUrlAttr(tagName, attrName, attrValue, windowLocation) {
   return attrValue;
 }
 
+/**
+ * Tag policy for handling what is valid html in templates.
+ * @type {!Function}
```

Comment:
Add `@param` and `@return` declarations. No need for `@type`.

Reply:
done. thanks

Reply:
tests passing. PTAL.

Reply:
Did you push these changes? Don't see them, though NBD to address in follow-up.